user_interface/EbikeUIController/can.cpp

```diff
@@ -0,0 +1,121 @@
+#include "can.h"
+
+// CAN message flags 
+bool CAN_MB0_BikeStatus_Flag = false;
```

Comment:
Please change the type of these to "static bool" so they are only global to this file. See other comments for how to provide the main file with read-only access to these variables